ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-path-selection.cfg

```diff
@@ -55,16 +55,16 @@ router path-selection
    load-balance policy LB-EMPTY
    !
    load-balance policy LB-P-1
-      hop count lowest
       loss-rate 17
+      hop count lowest
       path-group PG-5
       path-group PG-2 priority 42
       path-group PG-4 priority 42
       path-group PG-3 priority 666
    !
    load-balance policy LB-P-2
-      jitter 666
       latency 42
+      jitter 666
```

Comment:
Add '!' between lines: https://github.com/aristanetworks/avd/pull/4462/files#diff-085f1adc7526d32f716b1fc0568aa262cc43cf147861b6dab50b15629a3d0d62R84

Reply:
we can skip the exclamation mark sign verification for now